cmd/provider/announce.go

```diff
@@ -32,3 +43,36 @@ func announceCommand(cctx *cli.Context) error {
 	_, err = cctx.App.Writer.Write([]byte("Announced latest advertisement\n"))
 	return err
 }
+
+func announceHttpCommand(cctx *cli.Context) error {
+	indexer := cctx.String("indexer")
+
+	params := map[string][]byte{
+		"indexer": []byte(indexer),
+	}
+	bodyData, err := json.Marshal(&params)
+	if err != nil {
+		return err
+	}
+	body := bytes.NewBuffer(bodyData)
+
+	req, err := http.NewRequestWithContext(cctx.Context, http.MethodPost, adminAPIFlagValue+"/admin/announcehttp", body)
+	if err != nil {
+		return err
+	}
+	req.Header.Set("Content-Type", "application/json")
+
+	cl := &http.Client{}
+	resp, err := cl.Do(req)
+	if err != nil {
+		return err
+	}
+	defer resp.Body.Close()
+
+	if resp.StatusCode != http.StatusOK {
+		return errFromHttpResp(resp)
+	}
+
+	_, err = cctx.App.Writer.Write([]byte("Announced latest advertisement via HTTP\n"))
```

Comment:
maybe include the cid or announcement serialization?

Reply:
Responds with announced CID.